### PR TITLE
products listing page added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
product details

### CODE REVIEW

**Comment by Roshannair24** [comment]
Nice Ui, maintain the padding in your card, and center align the button.

**Review by Roshannair24 — APPROVED** [review]

**Comment by Digambar1411** [comment]
can align the card button in center, rest look fine

**Review by Digambar1411 — APPROVED** [review]

File: product-listing/products.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    overflow: hidden;
+    height: 89vh;
+    display: flex;
+}
+.sidebar{
+    overflow: auto;
+    padding: 0.5rem;
+    flex: 1 0;
+    flex-basis: 230px;
+    box-shadow: 1px 2px 2px 1px var(--secondary);
```
Comment (vijaytembugade) [inline product-listing/products.css:12]: you can create box-show variable, just suggestion.

**Review by vijaytembugade — APPROVED** [review]

File: product-listing/products.css (hunk 1 above)
Comment (vaibhav18matere) [reply product-listing/products.css:12]: sure, thanks!

**Pull request merged**